core/settings.py

```
@@ -235,3 +232,17 @@
 
+# Trusted proxies for password reset rate limiting client IP extraction
+TRUSTED_PROXY_IPS = [
+    ip.strip()
+    for ip in os.environ.get('TRUSTED_PROXY_IPS', '').split(',')
+    if ip.strip()
+]
+
+if IS_PRODUCTION and not TRUSTED_PROXY_IPS:
+    raise ImproperlyConfigured(
+        'TRUSTED_PROXY_IPS must be set in production'
+    )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)